bindings/c/test/mako/mako.c

```diff
@@ -290,13 +290,13 @@ int64_t run_op_getreadversion(FDBTransaction *transaction) {
   } while (err && retry--);
 
   if (err) {
-    fprintf(stderr, "ERROR: fdb_transaction_get_version: %s\n", fdb_get_error(err));
+    fprintf(stderr, "ERROR: fdb_transaction_get_read_version: %s\n", fdb_get_error(err));
     return -1;
   }
 
-  err = fdb_future_get_version(f, &rv);
+  err = fdb_future_get_int64(f, &rv);
```

Comment:
This one is a bit awkward because we can't change it until we've updated the API version of the test, and we can't change the API version of the test until we've updated the API version on the master branch. I'm thinking  it's probably best that we go ahead and do the API version update everywhere before finishing this PR.

Reply:
Sounds good. Defer this only a separate PR to update version to 620.

Reply:
Looking at the source of this a bit, it seems as if there is some intent to support multiple API versions. It may be good to follow that pattern here too, in that case.